modules/fluid_properties/doc/content/source/userobjects/TabulatedFluidProperties.md

```diff
@@ -60,13 +62,20 @@ and so on.
 Consider the example where TabulatedFluidProperties is used to reduce the cost of calculating
```

Comment:
Avoid referring to the class name since it's a base class name

Reply:
Actually, I think it'd be ok to do: a `TabulatedFluidProperties` object

Reply:
fixed